packages/@ember/-internals/glimmer/tests/integration/components/tracked-test.js

```diff
@@ -485,6 +485,55 @@ if (EMBER_METAL_TRACKED_PROPERTIES) {
           assert.equal(outerRenderCount, 2, 'outer component updates based on context');
           assert.equal(innerRenderCount, 3, 'inner component updates based on outer component');
         }
+
+        '@test computed properties computed properties can depend on args'() {
+          class TestComponent extends GlimmerishComponent {
+            @computed('args.text')
+            get text() {
+              return this.args.text;
+            }
+          }
+
+          this.registerComponent('test', {
+            ComponentClass: TestComponent,
+            template: '<p>{{this.text}}</p>',
+          });
+
+          this.render('<Test @text={{this.text}}/>', {
+            text: 'hello!',
+          });
+
+          this.assertText('hello!');
+
+          runTask(() => this.context.set('text', 'hello world!'));
+          this.assertText('hello world!');
+
+          runTask(() => this.context.set('text', 'hello!'));
+          this.assertText('hello!');
+        }
+
+        '@test named args are enumerable'() {
```

Comment:
Can you add another test for using `{{#each-in this.args as |arg value|}}`?

Reply:
Good thing we added that, turns out it is necessary to support symbols in the proxy. I decided to make the guard in the `get` trap a bit more generic to handle that.